components/ord-service/src/main/java/com/sap/cloud/cmp/ord/service/storage/model/EntityTypeMappingEntity.java

```diff
@@ -0,0 +1,58 @@
+package com.sap.cloud.cmp.ord.service.storage.model;
+
+import java.util.List;
+import java.util.UUID;
+
+import javax.persistence.CollectionTable;
+import javax.persistence.Column;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import org.eclipse.persistence.annotations.Convert;
+import org.eclipse.persistence.annotations.TypeConverter;
+
+import com.sap.cloud.cmp.ord.service.storage.model.helperentities.APIModelSelector;
+import com.sap.cloud.cmp.ord.service.storage.model.helperentities.EntityTypeTarget;
+import com.sap.olingo.jpa.metadata.core.edm.annotation.EdmIgnore;
+
+@Entity(name = "entityTypeMapping")
+@Table(name = "tenants_entity_type_mappings")
+public class EntityTypeMappingEntity {
+    @javax.persistence.Id
+    @Column(name = "id")
+    @Convert("uuidConverter")
+    @TypeConverter(name = "uuidConverter", dataType = Object.class, objectType = UUID.class)
+    private UUID Id;
+
```

Comment:
Add field for the tenant id:

    @EdmProtectedBy(name = "tenant_id")
    @EdmIgnore
    @Column(name = "tenant_id", length = 256)
    @Convert("uuidConverter")
    @TypeConverter(name = "uuidConverter", dataType = Object.class, objectType = UUID.class)
    private UUID tenant;